### PR TITLE
Changes 

### PR DESCRIPTION
**Pull request by vadrian89**
High guys, 

Please check / test and let me know if you approve of this, or let me know issues to fix so I can merge it with Daryl's branch so he can make a new package.

Thanks!

### CODE REVIEW

**Comment by kd8bny** [comment]
Roger will do!

**Comment by vadrian89** [comment]
Apparently I was calculating the speed from Open Weather Map wrong because I was setting the unit as mph (don't know so dont ask me why :P) even though they send the default speed unit as m/s , changed it now and will commit this evening.

Edit: 
I am also thinking of hidding the buttons for "Support" and "How To" until they are implementing so they don't stir confusion amongst users.

**Comment by kd8bny** [comment]
If you guys go ahead and build from a clean system (without cumulus install) do you recieve the following error?

"(cumulus:11030): GLib-GIO-ERROR **: Settings schema 'net.launchpad.cumulus' is not installed"

It isnt a huge deal you just have to recompile the glib schema. 
I think the issue is in the build file probably something I forgot to change awhile ago

ps the changes look great Adrian! Lets merge 'em!
I concur with hiding those 2 buttons too...Just till they do something

**Comment by vadrian89** [comment]
Yeah I think I had this error when trying to run on a clean system.

Btw I'm also looking into packaging/deploying for ubuntu but for today I'm
finished.
To much learning for today.

I'm at a point of making a new setup.py file / making a snap file / buildin
deb packages for ppa.
Something bugs me, what package should contain gi.repository?

Pe 25 sept. 2016 5:13 p.m., "Daryl Bennett" notifications@github.com a
scris:

**Comment by kd8bny** [comment]
What package of ours or what package gi.repository comes from?

**Comment by vadrian89** [comment]
What package is gi.repository coming from.

Here is my askubuntu question:
http://askubuntu.com/questions/829374/missing-dependency-in-python-application-packaged-with-snapcraft

2016-09-25 17:26 GMT+03:00 Daryl Bennett notifications@github.com:

**Comment by kd8bny** [comment]
I think this is what youre looking for 

https://wiki.gnome.org/Projects/PyGObject#Binaries

apt-get install python-gi

**Comment by vadrian89** [comment]
Ill look a bit more later, now I'm relaxing :)

Pe 25 sept. 2016 5:29 p.m., "Daryl Bennett" notifications@github.com a
scris:

**Comment by vadrian89** [comment]
Can i mail the setup.py so you guys have a look at it? And if needed it fix
it? :-)

Pe 25 sept. 2016 5:38 p.m., "Verban Adrian" verbanady@gmail.com a scris:

**Comment by kd8bny** [comment]
Of course haha...enjoy your  relax time.
Its morning here so I have all day lol

**Comment by vadrian89** [comment]
For me it 5:42 PM. :), I'll send it in 10 mins

Pe 25 sept. 2016 5:41 p.m., "Daryl Bennett" notifications@github.com a
scris:

**Comment by vadrian89** [comment]
Here is the setup.py I came up atm.

2016-09-25 17:43 GMT+03:00 Verban Adrian verbanady@gmail.com:

**Comment by vadrian89** [comment]
Made some progress yesterday evening, though I had little time (about 2 hours), I will continue this evening maybe I will manage to pull it through.  
Need to remake the tarball structure made from the setup.py, for a ppa I need to build it Ubuntu tools so debian documents need to be avoided. 

Anyway, the general ideea is that I'm making progress and if I don't manage to make a debian package for the ppa by the end of this day I will surely finish by the end of this week.
After I manage to succede with debian I will continue with snapcraft.

File: data/glib-2.0/schemas/com.github.cumulus.gschema.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <schemalist gettext-domain="cumulus">
-  <schema id="net.launchpad.cumulus" path="/net/launchpad/cumulus/">
+  <schema id="com.github.cumulus" path="/apps/cumulus/" gettext-domain="cumulus">
```
Comment (vadrian89) [inline data/glib-2.0/schemas/com.github.cumulus.gschema.xml]: need to remove one gettext-domain="cumulus" as it already was in schemalist, don't know how I missed it.

**Comment by archisman-panigrahi** [comment]
Hi, I am really sorry to have kept you waiting, but, I have ported Cumulus to Quickly. I need a few more days to set up License etc. before I can upload to the PPA but it is quite ready. I am using master branch of @vadrian89, and found a peculiar bug. Some opacity values are not as expected. They are 
o0.1
o0.15000000000000002
o0.2
o0.25
o0.30000000000000004
o0.35
o0.4
o0.45000000000000007
o0.5
o0.55
o0.6
o0.65
o0.7000000000000001
o0.75
o0.8
o0.85
o0.9
o0.9500000000000001
o1
instead of expected o0.15 etc. And, opacity of those values are not working as expected.
I tried to round values as described [here](http://stackoverflow.com/a/11832950/3128341), but that does not work.

screenshot from 2016-09-28 19 34 08

**Comment by vadrian89** [comment]
It's ok, just came home from work will look into the issue rigtht away.

Pe 28 sept. 2016 5:29 p.m., "Archisman Panigrahi" notifications@github.com
a scris:

**Comment by archisman-panigrahi** [comment]
Check this  https://launchpad.net/~cumulus-team/+archive/ubuntu/cumulus

**Comment by archisman-panigrahi** [comment]
@vadrian89 I just used Quickly template, changed necessary files and copied files from `data/media` of your master branch. System language has to be manually added to Python. Also, I have not tried your setup.py, and used the setup.py created by Quickly. We will hve to try to have one source code that will work with both Debian and Snap packaging system, or otherwise will have to maintain two branches.

PreferencesDialog.py was added by Quickly and probably, can safely be removed

**Comment by archisman-panigrahi** [comment]
I have updated setup.py (will push soon). @vadrian89 How to round opacity values upto 2 decimal places? I tried

```
function updateTitle(val) {
    document.title = "o" + Math.Round(val*100)/100;
    localStorage.app_opacity = Math.Round(val*100)/100;
}
```

But that just stops updating Title when opacity is change

**Comment by vadrian89** [comment]
Can you please mail me the setup.py file from Quickly?

Pe 29 sept. 2016 9:43 a.m., "Archisman Panigrahi" notifications@github.com
a scris:

**Comment by archisman-panigrahi** [comment]
I have fixed it. Will push the commit when this Pull request is accepted. I was doing `Math.Round` instead of `Math.round`, which created the problem

**Comment by vadrian89** [comment]
Ok, I will accept the pull request this evening after taking another look
into it, now I'm at work and a bit busy.

Pe 29 sept. 2016 10:35 a.m., "Archisman Panigrahi" notifications@github.com
a scris:

**Pull request merged**